src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -761,12 +796,16 @@ def get_devices(_sys_block_path='/sys/block'):
 
     device_facts = {}
 
-    block_devs = get_block_devs_lsblk()
+    block_devs = get_block_devs_sysfs(_sys_block_path)
```

Comment:
`def get_block_devs_lsblk()` L734 won't be used anywhere else after that, should we drop it and all related tests in `tests/util/test_disk.py` altogether?

Reply:
I'm fine with removing it unless there's a reason to keep it around. But if it's ever needed later it'll be in the repo's history.